package.json

```diff
@@ -5,6 +5,9 @@
   "repository": "microsoft/FluidHelloWorld",
   "license": "MIT",
   "author": "Microsoft",
+  "engines": {
+    "node": ">=14.19.0 <18.0.0"
```

Comment:
Should this target 18+? Node LTS is now 18.
Readme was updated to say 16+ in May.